net/remote/remote-node.sh

```diff
@@ -295,6 +317,12 @@ EOF
       args+=(--tpu-enable-udp)
     fi
 
+    if ${TPU_CLIENT}; then
+        args+=(--use-tpu-client)
+    elif ${RPC_CLIENT}; then
+        args+=(--use-rpc-client)
+    fi
+
```

Comment:
This doesn't seem like the right place. Shouldn't these changes be in `remote-client.sh`? https://github.com/solana-labs/solana/blob/daf71003d30c9e3dac48b4391e03c33025b88c61/net/remote/remote-client.sh#L49
Also, might be more helpful if you handle adding the `--url` parameter and remove `--entrypoint` for these client types

Reply:
Done - moved the changes to the appropriate place, though IMHO the --url parameter-related changes should be saved for a separate PR.

Reply:
> IMHO the --url parameter-related changes should be saved for a separate PR.

Please include it in this one. You'll see why if you actually try to run either default:
```
 ./net.sh restart --client-type <rpc-client|tpu-client>
```